demos/common/python/openvino/model_zoo/model_api/adapters/ovms_adapter.md

```diff
@@ -0,0 +1,70 @@
+
+## Prerequisites
+
+`OVMSAdapter` enables inference via gRPC calls to OpenVINO Model Server, so in order to use it you need two things:
+- OpenVINO Model Server that serves your model
+- [`ovmsclient`](https://pypi.org/project/ovmsclient/) package installed to enable communication with the model server
+
+### Deploy OpenVINO Model Server
+
+Model Server is distributed as a docker image, so you can pull it from Docker Hub with:
+
+```
+docker pull openvino/model_server
+```
+
+See [model server documentation](https://github.com/openvinotoolkit/model_server/blob/main/docs/docker_container.md) to learn how to deploy OpenVINO optimized models with OpenVINO Model Server.
+
+### Install ovmsclient
+
+`ovmsclient` package is distributed on PyPi, so the easiest way to install it is via:
+
+```
+pip install ovmsclient
+```
+
+## Model configuration
+
+When using OpenVINO Model Server model cannot be directly accessed from the client application (like OMZ demos). Therefore any configuration must be done on model server side.
+
+### Input reshaping
+
+For some use cases you may want your model to reshape to match input of certain size. In that case, you should provide `--shape auto` parameter to model server startup command. With that option, model server will reshape model input on demand to match the input data.
+
+### Inference options
+
+It's possible to configure inference related options for the model in OpenVINO Model Server with options:
+- `--target_device` - name of the device to load the model to
+- `--nireq` - number of InferRequests
+- `--plugin_config` - configuration of the device plugin
+
+See [model server configuration parameters](https://github.com/openvinotoolkit/model_server/blob/main/docs/docker_container.md#configuration-parameters) for more details.
+
+### Example OVMS startup command
+```
+docker run -d --rm -v <models_repository>:/models -p 9000:9000 openvino/model_server:latest \
+--model_path <path_to_model> --model_name <model_name> --port 9000 --shape auto --nireq 32 --target_device CPU --plugin_config "{\"CPU_THROUGHPUT_STREAMS\": \"CPU_THROUGHPUT_AUTO\"}"
+```
+
+> **Note**: In demos, while using `--adapter ovms`, inference options like: `-nireq`, `-nstreams` `-nthreads` as well as device specification with `-d` will be ignored.
+
+## Running demos with OVMSAdapter
+
+To run the demo with model served in OpenVINO Model Server, you would have to provide `--adapter ovms` option and modify `-m` parameter to indicate model inference service instead of the model files. Model parameter for `OVMSAdapter` follows this schema:
+
+```<service_address>/models/<model_name>[:<model_option>]```
+
+- `<service_address>` - OVMS gRPC service address in form `<address>:<port>`
+- `<model_name>` - name of the target model (the one specified by `model_name` parameter in the model server startup command)
+- `<model_version>` *(optional)* - version of the target model (default: latest)
```

Comment:
You described `<model_version>`, which is absent in `<service_address>/models/<model_name>[:<model_option>]` and didn't decribe `<model_option>`. It's a bit confusing. 
Please, specify where `<model_version>` should be and provide more detailed explanation for `<model_option>`.

Reply:
Thank you for paying your attention!
It seems the correct schema is `<service_address>/models/<model_name>[:<model_version>]`, there `model_option` is not provided.
@mzegla, please, update the schema with a correct name. In addition, it would be great to extend the following example of setting `-m` value with some model_version and describe it, like `localhost:9000/models/my_model:1`

Reply:
Good catch. My mistake, I meant `model_version`, but somehow I put `model_option` there.